api/lightwallet_rest.md

```diff
@@ -76,9 +82,12 @@ Information needed to spend an output.
 > `tx_hash` and `tx_prefix_hash` are determined by how `monerod` computes the
 > hash.
 
-> `rct` is the concatenation of the public commitment, then the decrypted
-> ringct mask value, and finally the decrypted ringct amount value. The value
-> is returned as a single 96-byte binary blob.
+> `rct` is, for ringct outputs, a 96-byte blob containing the concatenation of
+> the public commitment, then the ringct mask value, and finally the ringct
+> amount value. For ringct coinbase outputs, the mask is always the identity
```

Comment:
This comment got lost I think ...

There are no "other ringct outputs", this object is strictly for spending an output. The `amount` field would have to be updated otherwise; ringct would always be zero for "other ringct outputs".

Also, the ringct values are always encrypted.

Reply:
"Other ringct outputs" here means other than the "ringct coinbase outputs" mentioned earlier, i.e. non-coinbase ringct outputs. I guess I should just write exactly that, for clarity.

For the coinbase txes, the server is now to construct a special rct object consisting of the 
_unencrypted_ values `rct::identity()` for the mask and `0` for the amount. Previously, the client was [unable to spend ringct coinbase outputs](https://github.com/mymonero/mymonero-app-js/issues/240) at all. This was the solution chosen [therein](https://github.com/mymonero/mymonero-app-js/issues/240#issuecomment-436352783), not least because the OpenMonero server already does it this way. 

Does this make sense now, or do you think something (other than `s/other/non-coinbase/`) still needs to be changed?